dev-tools/packaging/templates/ironbank/auditbeat/hardening_manifest.yaml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: v1
+
+# The repository name in registry1, excluding /ironbank/
+name: "elastic/beats/auditbeat"
+
+# List of tags to push for the repository in registry1
+# The most specific version should be the first tag and will be shown
+# on ironbank.dsop.io
+tags:
+  - "{{ beat_version }}"
+  - "latest"
+
+# Build args passed to Dockerfile ARGs
+args:
+  BASE_IMAGE: "redhat/ubi/ubi8"
+  BASE_TAG: "8.6"
+  ELASTIC_STACK: "{{ beat_version }}"
```

Comment:
I think this may be from the current build method. Forgive me if this is somewhere and I'm just not seeing it, but is this value going to be present in the new build system that you're proposing?

Reply:
These files are stored in the `templates` folder, so the mage build system in Beats provides some templating interpolation, so `"{{ beat_version }}"` will be replaced with the current version (see https://github.com/elastic/beats/pull/32251/files#diff-01c144140f91f71381dc02e119a1ab1dce62512ce9bcf72745c8d55a125158f7R168 L168 of the `dev-tools/mage/pkg.go`